dangerzone/isolation_provider/container.py

```diff
@@ -3,8 +3,8 @@
 import logging
 import os
 import pathlib
-import pipes
 import platform
+import shlex
```

Comment:
`from shlex import quote`
alternative way

Reply:
I like keeping the namespace, so `shlex.quote` is fine :-)